README.md

```diff
@@ -48,6 +48,7 @@ welcomed.
 ## Maintainers:
 
 * Jeremy Avigad (@avigad): analysis
+* Anne Baanen (@Vierkantor): algebra, number theory
```

Comment:
```suggestion
* Anne Baanen (@Vierkantor): algebra, number theory, tactics
```
@Vierkantor you can accept the suggestion if you want or keep it as is.

Reply:
You were ahead of me by 13 seconds :)